source/features/open-all-conversations.tsx

```diff
@@ -13,15 +14,12 @@ function getUrlFromItem(issue: Element): string {
 		.href;
 }
 
-// eslint-disable-next-line import/prefer-default-export
-export function confirmOpen(count: number): boolean {
-	return count < 10 || confirm(`This will open ${count} new tabs. Continue?`);
-}
+const issueListSelector = pageDetect.isGlobalConversationList()
+	? '#js-issues-toolbar div'
+	: 'div[aria-label="Issues"][role="group"]';
 
 function onButtonClick(): void {
-	const modifier = pageDetect.isGlobalConversationList() ? '' : ' + div ';
-	const issues = select.all(`#js-issues-toolbar:not(.triage-mode) ${modifier} .js-issue-row`);
```

Comment:
The `.triage-mode` part of the selector was a vestigial residue from the previous incarnation of the feature (`batch-open-conversations`). Now we don't really care whether some issues are checked or not.

Reply:
Me again 2 minutes ago https://github.com/refined-github/refined-github/issues/4085#issuecomment-949492199 😂